packages/x-components/static-docs/experience-search-and-discovery/recommendations.md

```diff
@@ -10,40 +10,89 @@ tags:
 
-::: interact
+<CardCarousel :cards="[
+    'ui_ref/recommendations',
+    'ui_ref/semantics',
+    'ui_ref/preview',
+    'ui_ref/nq'
+    ]"
+/>
 
-Can't quite capture the concept? Learn more about
-[Recommendations](/explore-empathy-platform/features/recommendations-overview.md).
+::: warning
+
+For correct performance, make sure that your current search service supports
+this type of features.
 
 :::
 
 ## Tailor the web experience
+Based on your business strategy and your shoppers’ needs, the Interface X Components library offers the following configuration options and properties:
 
-- Configure the position and place it wherever you prefer, although they usually appear on the
-  results page.
-- Show as many recommendations as you want.
+::: details Top Products UI experience
+Use the Recommendations module to configure the layout of Top Products in your commerce search UI. You can:
+- Configure the position and place it wherever you prefer, although it commonly appears on the SERP, among the product results.
+- Show as many product recommendations as you want.
 - Animate the display of the component at your ease.
 - Customize content. Show whatever you need: text, images, icons.
-- Extend the performance with results-related components.
+- Extend the performance with result-related components.
+
+:::
+
+
+::: details Vectorized Recommendations UI experience    
+
+Use the Semantics Queries module to configure the layout of Vectorized Recommendations in your commerce search UI. You can:
+- Configure the position and place it wherever you prefer, although it usually appears on the SERP, among the product results.
+- Show as many product recommendations as you want. Display the total number of recommendations shown.
+- Use different layouts: carousel, list, grid. Although, a carousel or horizontal slider panel is the preferred layout.
+- Animate the display of the component at your ease.
+- Customize content. Although, the name of product recommendations is shown by default.
+- Extend the performance with query preview-related components to display related content, such as the suggested query, product result images, and links.
+
+::: 
+
+
+::: details Brand Recommendations UI experience  
+
+Use the Queries Preview module to configure the layout of Brand Recommendations in your commerce search UI. You can:
+- Configure the position and place it wherever you prefer, although it usually appears at the pre-search stage.
+- Display the suggested queries and the total number of recommendations shown.
+- Use different layouts: carousel, list, grid. Although, a carousel or horizontal slider panel is the preferred layout.
+- Animate the display of the component at your ease.
+- Customize content. Although, the name of product recommendations is shown by default.
+- Extend the performance with result-related components to display related content, such as product result images and links.
```

Comment:
Having said this a couple of bullets before:

Use different layouts: carousel, list, grid. Although, a carousel or horizontal slider panel is the preferred layout.

What is the intention of this one?